### PR TITLE
Add a guide for creating documentation images

### PR DESCRIPTION
**Pull request by skyace65**
Supersedes #6333. Page organization changes broke that one.

Creates a dedicated page in the contributing section for adding documentation images. This page acts as a basic guide for taking screenshots, converting to webp if necessary, how to use Krita to crop or resize images, and adding images to the documentation. This page also writes out several standards:

    Screenshots should be created from a 1080p screen or scaled down
    Outlines and arrows should use the color yellow (multiple outlines are an exception but avoid red)
    Outline should be 2 pixels wide (it's what I've been doing)
    Text should be made with a simple easy to read font (obvious, but probably best to have it written down)

This also changes the standard for image format from png to webp given recent discussions in rocket chat. I know there was talk about doing lossy webp compression for images over a certain size, what do we want to make the cutoff?

Lastly, when I started making this PR I was going to add a full tutorial for creating outlines in Inkscape. That section quickly became huge. If people want I'm willing to write it but I'm not sure if we want to add something that large to the page especially when community tutorials exist.

Closes https://github.com/godotengine/godot-docs/issues/6173

### CODE REVIEW

**Comment by Calinou** [comment]
> This also changes the standard for image format from webp to png given recent discussions in rocket chat.

Don't you mean the opposite (changing from PNG to WebP)?

**Comment by skyace65** [comment]
> > This also changes the standard for image format from webp to png given recent discussions in rocket chat.
> 
> Don't you mean the opposite (changing from PNG to WebP)?

Yeah, my bad

File: contributing/documentation/docs_image_guidelines.rst [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+Each Linux desktop environment has it's own screenshot tool. For example,
+on KDE Plasma the program Spectacle is used for taking screenshots. If your
+distribution doesn't come with one by default try searching its package
+repository, or Flathub if that's supported.
+
+All screenshots should ideally be taken on a 1080p screen. Anything higher
+resolution is adding detail that doesn't make the documentation better and
+dramatically increases file size. If you're taking screenshots on a higher
+resolution screen the screenshot should be scaled down. There are instructions
+on  how to do this later in this page.
```
Comment (mhilbrunner) [inline contributing/documentation/docs_image_guidelines.rst:34]:
```suggestion
on  how to do this later on this page.
```

**Comment by skyace65** [comment]
Fixed.

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you! 🎉 Merged.

**Pull request merged**